parts/footer-alpha-v3.html

```diff
@@ -0,0 +1,83 @@
+<!-- wp:group {"style":{"border":{"top":{"color":"var:preset|color|secondary-brand","width":"4px"},"right":{"width":"0px","style":"none"},"bottom":{"color":"var:preset|color|secondary-brand","width":"4px"},"left":{"width":"0px","style":"none"}},"spacing":{"padding":{"right":"2rem","left":"2rem","top":"2rem","bottom":"2rem"}}},"backgroundColor":"nonary","textColor":"white","layout":{"type":"constrained","contentSize":"1330px"},"metadata":{"name":"Territory acknowledgement"}} -->
+<div class="wp-block-group has-white-color has-nonary-background-color has-text-color has-background" style="border-top-color:var(--wp--preset--color--secondary-brand);border-top-width:4px;border-right-style:none;border-right-width:0px;border-bottom-color:var(--wp--preset--color--secondary-brand);border-bottom-width:4px;border-left-style:none;border-left-width:0px;padding-top:2rem;padding-right:2rem;padding-bottom:2rem;padding-left:2rem"><!-- wp:paragraph {"style":{"typography":{"fontSize":"1rem"}}} -->
+	<p style="font-size:1rem">The BC Public Service acknowledges the territories of First Nations around B.C. and is grateful to carry out our work on these lands. We acknowledge the rights, interests, priorities and concerns of all Indigenous Peoples—First Nations, Métis and Inuit—respecting and acknowledging their distinct cultures, histories, rights, laws and governments.</p>
```

Comment:
just curious where this text came from, and why it should be the default (i like it, just want to know the logic behind it)

Reply:
This text came from the Alpha 3 update Figma attached to the ticket as part of the design system layout: https://www.figma.com/file/7qqTHUz3u9nYN17Tdwfvg3/Alpha-3.0-updates?node-id=1%3A6415&mode=dev